en/docs/assets/js/theme.js

```diff
@@ -0,0 +1,171 @@
+/*!
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
shall we update the year to 2024

Reply:
Fixed in daad86212fb7a2e8abec9643acd86557eae0078b.